lib/image/color.ex

```diff
@@ -147,6 +147,14 @@ defmodule Image.Color do
     [r, g, b, a]
   end
 
+  def rgba_color!(color, a)
+  when (is_binary(color) or is_atom(color)) and is_integer(a) do
+    a = Integer.mod(a, 256)
```

Comment:
Not sure whether this makes sense.
As I mentioned, I am not really sure whether the alpha value should strictly be between [0.0, 1.0] or [0, 255].